docs/navigating-intended.md

```diff
@@ -29,6 +29,147 @@ or
 {% endfor %}
 ```
 
+## Provided Jinja2 Filters
+
+This Plugin provides three Jinja2 filters to aid in generating configs.
+The `find_available_files` filter is provided to identify full filenames based on
+the end of filenames and extension.
+The `collect_preferred_file` and `collect_preferred_files` are provided to identify the
```

Comment:
This is similar to the Ansible construct, `with_first_found`, does it make sense to continue similarly? 

`first_found_file` and `first_found_files`?